src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -734,6 +772,16 @@ export class ConnectionWidget extends lifecycle.Disposable {
 				this._tableContainer.classList.add('hide-azure-accounts');
 			}
 
+			if (this._customOptionWidgets) {
+				this._customOptionWidgets.forEach((widget, i) => {
+					if (widget instanceof SelectBox) {
+						(widget as SelectBox).selectWithOptionName(this.getModelValue(connectionInfo.options[this._customOptions[i].name]));
```

Comment:
You should be able to remove the cast now - type narrowing should already make this known to be a SelectBox